### PR TITLE
add docker file for test container

### PR DESCRIPTION
**Pull request by b97pla**
Adds a docker container that can bounce requests from `ngi_uu_workflow`

### CODE REVIEW

File: files/Dockerfile.tester [hunk 1]
```diff
@@ -0,0 +1,72 @@
+FROM nginx:latest
+
+COPY tests/test_data/runfolders /data/tester/runfolders
```
Comment (matrulda) [inline files/Dockerfile.tester:3]: Does the source refer to the snpseq_packs repo? Do we have minimal runfolders there?

File: docker-compose.yml [hunk 2]
```diff
@@ -267,7 +271,16 @@ services:
       - private
     volumes:
       - stackstorm-redis:/data
-
+  tester:
```
Comment (matrulda) [inline docker-compose.yml:274]: Could we call it snpseq-tester or something? So that it is clear that it is something we have added.

**Review by matrulda — COMMENTED** [review]
Nice work! Looks good to me, left you a comment and have a general one: Where do you plan to document how to use this feature, snpseq_packs?

File: files/Dockerfile.tester (hunk 1 above)
Comment (b97pla) [reply files/Dockerfile.tester:3]: yes, that's the context specified in `docker_compose.yml`. I have a PR coming with a mock runfolder (and instructions)

File: docker-compose.yml (hunk 2 above)
Comment (b97pla) [reply docker-compose.yml:274]: good suggestion, I'll fix

**Comment by b97pla** [comment]
thank for the review! 🙏 

I have a branch in `snpseq_packs` with instructions (and some documentation will probably end up in confluence). I need this merged before I can open that PR though

**Comment by matrulda** [comment]
Awesome :+1:

**Review by matrulda — APPROVED** [review]

**Pull request merged**